lms/static/js/learner_dashboard/views/collection_list_view.js

```diff
@@ -1,68 +1,53 @@
-        function(Backbone,
-                  $,
-                  _,
-                  gettext,
-                  StringUtils,
-                  HtmlUtils,
-                  emptyProgramsListTpl) {
-            return Backbone.View.extend({
-
-                initialize: function(data) {
-                    this.childView = data.childView;
-                    this.context = data.context;
-                    this.titleContext = data.titleContext;
-                },
-
-                render: function() {
-                    var childList;
-
-                    if (!this.collection.length) {
-                        if (this.context.marketingUrl) {
-                            // Only show the advertising panel if the link is passed in
-                            HtmlUtils.setHtml(this.$el, HtmlUtils.template(emptyProgramsListTpl)(this.context));
-                        }
-                    } else {
-                        childList = [];
-
-                        this.collection.each(function(model) {
-                            var child = new this.childView({
-                                model: model,
-                                context: this.context
-                            });
-                            childList.push(child.el);
-                        }, this);
-
-                        if (this.titleContext) {
-                            this.$el.before(HtmlUtils.ensureHtml(this.getTitleHtml()).toString());
-                        }
-
-                        this.$el.html(childList);
-                    }
-                },
-
-                getTitleHtml: function() {
-                    var titleHtml = HtmlUtils.joinHtml(
-                        HtmlUtils.HTML('<'),
-                        this.titleContext.el,
-                        HtmlUtils.HTML(' class="sr-only collection-title">'),
-                        StringUtils.interpolate(this.titleContext.title),
-                        HtmlUtils.HTML('</'),
-                        this.titleContext.el,
-                        HtmlUtils.HTML('>'));
-                    return titleHtml;
-                }
-            });
-        }
-    );
-}).call(this, define || RequireJS.define);
+import Backbone from 'backbone';
+
+import HtmlUtils from 'edx-ui-toolkit/js/utils/html-utils';
+import StringUtils from 'edx-ui-toolkit/js/utils/string-utils';
+
+import emptyProgramsListTpl from '../../../templates/learner_dashboard/empty_programs_list.underscore';
+
+class CollectionListView extends Backbone.View {
+  initialize(data) {
+    this.childView = data.childView;
+    this.context = data.context;
+    this.titleContext = data.titleContext;
+  }
+
+  render() {
+    if (!this.collection.length) {
+      if (this.context.marketingUrl) {
+        // Only show the advertising panel if the link is passed in
+        HtmlUtils.setHtml(this.$el, HtmlUtils.template(emptyProgramsListTpl)(this.context));
+      }
+    } else {
+      const childList = [];
```

Comment:
Are we still supposed to define variables (or const's) in the top of the function in es6?

Reply:
Naw, es6 don't care where.